.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+##  Here is what I would like to see new or change within PokeNurse
+
+
```

Comment:
I'm all for it @vinnymac but to wait for me to come up with the words is a long process ;-)

It'd be faster to merge this to develop, add content, then bring to master. Tweaking along the way is a given.